lib/TAP/Formatter/Color.pm

```diff
@@ -12,7 +12,7 @@ my $NO_COLOR;
 BEGIN {
     $NO_COLOR = 0;
 
-    eval 'use Term::ANSIColor';
+    eval 'use Term::ANSIColor qw//';
```

Comment:
We don't need to soften the prereq.  We don't need the eval.  Term::ANSIColor is core since 5.6.